### ISSUE TITLE
Missing and new architecture values

### ISSUE THREAD

**Issue by marchof** [foojayio/discoapi#48]
Since yesterday there are packages with an empty `architecture` tag or new values not seen so far (`mipsel`, `armhf`, `armel`). Is this intended?

```
{"id":"cf8f6d3ee66aa3b92d00f2f6eefc8165","archive_type":"tar.gz","distribution":"dragonwell","major_version":8,"java_version":"8.0.275+2","distribution_version":"8.5.5","feature_version":8,"interim_version":0,"update_version":275,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":true,"filename":"Alibaba_Dragonwell_8.5.5-FP1_source.tar.gz","ephemeral_id":"88da2bcb9ea69bc460cc9cd9870c290dcd8fbdb9","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/88da2bcb9ea69bc460cc9cd9870c290dcd8fbdb9","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/88da2bcb9ea69bc460cc9cd9870c290dcd8fbdb9/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"fea5becb60519734547143224a738bc6","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"mipsel","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jdk_8u302-b08-1_mipsel.deb","ephemeral_id":"84a45f290e90466c484e2d02ab9b2ee6b918113c","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/84a45f290e90466c484e2d02ab9b2ee6b918113c","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/84a45f290e90466c484e2d02ab9b2ee6b918113c/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"8437424075a0b2e3d1f7b158b57dc46c","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armhf","fpu":"hard_float","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jdk_8u302-b08-1_armhf.deb","ephemeral_id":"cfbe50d570c0c5f4fea4a7d40b706ac36f9f84ec","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/cfbe50d570c0c5f4fea4a7d40b706ac36f9f84ec","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/cfbe50d570c0c5f4fea4a7d40b706ac36f9f84ec/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"691c858bea4ae2842d80cca7e0325715","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armhf","fpu":"hard_float","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jre_8u302-b08-1_armhf.deb","ephemeral_id":"43303ed61aab3e6532f4a27c381d0366fbffc7c8","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/43303ed61aab3e6532f4a27c381d0366fbffc7c8","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/43303ed61aab3e6532f4a27c381d0366fbffc7c8/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"a28101ab170f2b067806a1c8c31fba5b","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armel","fpu":"soft_float","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jre_8u302-b08-1_armel.deb","ephemeral_id":"0d9bf538e620d5308d2fe330d11e33d08a2acf14","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/0d9bf538e620d5308d2fe330d11e33d08a2acf14","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/0d9bf538e620d5308d2fe330d11e33d08a2acf14/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"530444b97325882d18032a2fcc68405f","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"mipsel","fpu":"unknown","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jre_8u302-b08-1_mipsel.deb","ephemeral_id":"79396ff9bd1116a4b246db0fcd5376ae6a535029","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/79396ff9bd1116a4b246db0fcd5376ae6a535029","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/79396ff9bd1116a4b246db0fcd5376ae6a535029/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"31846e9f00471bfb81fc4bb071ecb663","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armel","fpu":"soft_float","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jdk_8u302-b08-1_armel.deb","ephemeral_id":"36ba32fcd98ebf29157e226b4826854c4a67c88c","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/36ba32fcd98ebf29157e226b4826854c4a67c88c","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/36ba32fcd98ebf29157e226b4826854c4a67c88c/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"8532d86aa95e74fe6f90505bbd181976","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"mipsel","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jdk_11.0.13+8-1_mipsel.deb","ephemeral_id":"112d941c0f2e2d38ea57d4bc623c1852400fc6dd","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/112d941c0f2e2d38ea57d4bc623c1852400fc6dd","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/112d941c0f2e2d38ea57d4bc623c1852400fc6dd/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"c12515edc056edfe6ad9c277241f447c","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armel","fpu":"soft_float","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jdk_11.0.13+8-1_armel.deb","ephemeral_id":"fca8ae3967be9a7d6410611fa449b638407b9186","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/fca8ae3967be9a7d6410611fa449b638407b9186","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/fca8ae3967be9a7d6410611fa449b638407b9186/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"ae7a87068e4e7024f0e91393407d375f","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"mipsel","fpu":"unknown","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jre_11.0.13+8-1_mipsel.deb","ephemeral_id":"da7724801cf16465b15639ec8361793e45793149","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/da7724801cf16465b15639ec8361793e45793149","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/da7724801cf16465b15639ec8361793e45793149/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"1bf2161b89f256442c497cfb12d9c138","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armel","fpu":"soft_float","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jre_11.0.13+8-1_armel.deb","ephemeral_id":"684b5021468a4854b6ed69f5c23c3c452b8ef4d3","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/684b5021468a4854b6ed69f5c23c3c452b8ef4d3","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/684b5021468a4854b6ed69f5c23c3c452b8ef4d3/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"d1f2f5634d910384e534ebd848b0784d","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armhf","fpu":"hard_float","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jre_11.0.13+8-1_armhf.deb","ephemeral_id":"67bff8bca3f02c2178b33de38999e1f53fa98926","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/67bff8bca3f02c2178b33de38999e1f53fa98926","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/67bff8bca3f02c2178b33de38999e1f53fa98926/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"39463a29540b8d47bcaeac38e6869b31","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armhf","fpu":"hard_float","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jdk_11.0.13+8-1_armhf.deb","ephemeral_id":"504a182d3b5392c2ade6a52341bed12cceb33ccb","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/504a182d3b5392c2ade6a52341bed12cceb33ccb","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/504a182d3b5392c2ade6a52341bed12cceb33ccb/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"7aa5c2c2f61d53024a75940b93b81d80","archive_type":"tar.gz","distribution":"dragonwell","major_version":11,"java_version":"11+9","distribution_version":"11.0.7.2","feature_version":11,"interim_version":0,"update_version":0,"patch_version":0,"build_version":9,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":true,"filename":"Alibaba_Dragonwell_11.0.7.2+9_source.tar.gz","ephemeral_id":"1e072b91c478dc3159679f960604c50b1fee40b1","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/1e072b91c478dc3159679f960604c50b1fee40b1","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/1e072b91c478dc3159679f960604c50b1fee40b1/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"cf8f6d3ee66aa3b92d00f2f6eefc8165","archive_type":"tar.gz","distribution":"dragonwell","major_version":8,"java_version":"8.0.275+2","distribution_version":"8.5.5","feature_version":8,"interim_version":0,"update_version":275,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":true,"filename":"Alibaba_Dragonwell_8.5.5-FP1_source.tar.gz","ephemeral_id":"2458e2884627cda0449515f9583b3c2a5fce9b4f","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/2458e2884627cda0449515f9583b3c2a5fce9b4f","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/2458e2884627cda0449515f9583b3c2a5fce9b4f/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"7aa5c2c2f61d53024a75940b93b81d80","archive_type":"tar.gz","distribution":"dragonwell","major_version":11,"java_version":"11+9","distribution_version":"11.0.7.2","feature_version":11,"interim_version":0,"update_version":0,"patch_version":0,"build_version":9,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":true,"filename":"Alibaba_Dragonwell_11.0.7.2+9_source.tar.gz","ephemeral_id":"1e072b91c478dc3159679f960604c50b1fee40b1","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/1e072b91c478dc3159679f960604c50b1fee40b1","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/1e072b91c478dc3159679f960604c50b1fee40b1/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"cf8f6d3ee66aa3b92d00f2f6eefc8165","archive_type":"tar.gz","distribution":"dragonwell","major_version":8,"java_version":"8.0.275+2","distribution_version":"8.5.5","feature_version":8,"interim_version":0,"update_version":275,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":true,"filename":"Alibaba_Dragonwell_8.5.5-FP1_source.tar.gz","ephemeral_id":"88da2bcb9ea69bc460cc9cd9870c290dcd8fbdb9","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/88da2bcb9ea69bc460cc9cd9870c290dcd8fbdb9","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/88da2bcb9ea69bc460cc9cd9870c290dcd8fbdb9/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"fea5becb60519734547143224a738bc6","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"mipsel","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jdk_8u302-b08-1_mipsel.deb","ephemeral_id":"84a45f290e90466c484e2d02ab9b2ee6b918113c","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/84a45f290e90466c484e2d02ab9b2ee6b918113c","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/84a45f290e90466c484e2d02ab9b2ee6b918113c/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"8437424075a0b2e3d1f7b158b57dc46c","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armhf","fpu":"hard_float","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jdk_8u302-b08-1_armhf.deb","ephemeral_id":"cfbe50d570c0c5f4fea4a7d40b706ac36f9f84ec","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/cfbe50d570c0c5f4fea4a7d40b706ac36f9f84ec","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/cfbe50d570c0c5f4fea4a7d40b706ac36f9f84ec/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"691c858bea4ae2842d80cca7e0325715","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armhf","fpu":"hard_float","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jre_8u302-b08-1_armhf.deb","ephemeral_id":"43303ed61aab3e6532f4a27c381d0366fbffc7c8","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/43303ed61aab3e6532f4a27c381d0366fbffc7c8","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/43303ed61aab3e6532f4a27c381d0366fbffc7c8/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"a28101ab170f2b067806a1c8c31fba5b","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armel","fpu":"soft_float","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jre_8u302-b08-1_armel.deb","ephemeral_id":"0d9bf538e620d5308d2fe330d11e33d08a2acf14","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/0d9bf538e620d5308d2fe330d11e33d08a2acf14","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/0d9bf538e620d5308d2fe330d11e33d08a2acf14/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"530444b97325882d18032a2fcc68405f","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"mipsel","fpu":"unknown","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jre_8u302-b08-1_mipsel.deb","ephemeral_id":"79396ff9bd1116a4b246db0fcd5376ae6a535029","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/79396ff9bd1116a4b246db0fcd5376ae6a535029","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/79396ff9bd1116a4b246db0fcd5376ae6a535029/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"31846e9f00471bfb81fc4bb071ecb663","archive_type":"deb","distribution":"debian","major_version":8,"java_version":"8.0.302+8","distribution_version":"8.0.302","feature_version":8,"interim_version":0,"update_version":302,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armel","fpu":"soft_float","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-8-jdk_8u302-b08-1_armel.deb","ephemeral_id":"36ba32fcd98ebf29157e226b4826854c4a67c88c","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/36ba32fcd98ebf29157e226b4826854c4a67c88c","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/36ba32fcd98ebf29157e226b4826854c4a67c88c/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"8532d86aa95e74fe6f90505bbd181976","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"mipsel","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jdk_11.0.13+8-1_mipsel.deb","ephemeral_id":"e6227306981a7b4eaa7ff846994ef218e5b42ce8","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/e6227306981a7b4eaa7ff846994ef218e5b42ce8","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/e6227306981a7b4eaa7ff846994ef218e5b42ce8/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"c12515edc056edfe6ad9c277241f447c","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armel","fpu":"soft_float","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jdk_11.0.13+8-1_armel.deb","ephemeral_id":"b3cb0272a1bb9df6159153d8bdfe33e32ce066bc","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/b3cb0272a1bb9df6159153d8bdfe33e32ce066bc","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/b3cb0272a1bb9df6159153d8bdfe33e32ce066bc/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"ae7a87068e4e7024f0e91393407d375f","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"mipsel","fpu":"unknown","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jre_11.0.13+8-1_mipsel.deb","ephemeral_id":"729588309248eb26185e28a8257abcb459b62494","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/729588309248eb26185e28a8257abcb459b62494","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/729588309248eb26185e28a8257abcb459b62494/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"1bf2161b89f256442c497cfb12d9c138","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armel","fpu":"soft_float","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jre_11.0.13+8-1_armel.deb","ephemeral_id":"f3a54c8ccffeb3c5593003b9502de670f5e2a1a8","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/f3a54c8ccffeb3c5593003b9502de670f5e2a1a8","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/f3a54c8ccffeb3c5593003b9502de670f5e2a1a8/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"d1f2f5634d910384e534ebd848b0784d","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armhf","fpu":"hard_float","package_type":"jre","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jre_11.0.13+8-1_armhf.deb","ephemeral_id":"bcae24c3351d53219adbab8805b486a46b166563","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/bcae24c3351d53219adbab8805b486a46b166563","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/bcae24c3351d53219adbab8805b486a46b166563/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"39463a29540b8d47bcaeac38e6869b31","archive_type":"deb","distribution":"debian","major_version":11,"java_version":"11.0.13","distribution_version":"11.0.13","feature_version":11,"interim_version":0,"update_version":13,"patch_version":0,"build_version":0,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"armhf","fpu":"hard_float","package_type":"jdk","javafx_bundled":false,"directly_downloadable":false,"filename":"openjdk-11-jdk_11.0.13+8-1_armhf.deb","ephemeral_id":"566f4fe5b99dd7dceea512bdd4989367f21674a6","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/566f4fe5b99dd7dceea512bdd4989367f21674a6","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/566f4fe5b99dd7dceea512bdd4989367f21674a6/redirect"},"free_use_in_production":true,"feature":[]}
{"id":"7aa5c2c2f61d53024a75940b93b81d80","archive_type":"tar.gz","distribution":"dragonwell","major_version":11,"java_version":"11+9","distribution_version":"11.0.7.2","feature_version":11,"interim_version":0,"update_version":0,"patch_version":0,"build_version":9,"latest_build_available":true,"release_status":"ga","term_of_support":"lts","operating_system":"linux","lib_c_type":"glibc","architecture":"","fpu":"unknown","package_type":"jdk","javafx_bundled":false,"directly_downloadable":true,"filename":"Alibaba_Dragonwell_11.0.7.2+9_source.tar.gz","ephemeral_id":"ef609189b54b204898d0ac5ddb7cc567f7df6ac6","links":{"pkg_info_uri":"https://api.foojay.io/disco/v2.0/ephemeral_ids/ef609189b54b204898d0ac5ddb7cc567f7df6ac6","pkg_download_redirect":"https://api.foojay.io/disco/v2.0/ephemeral_ids/ef609189b54b204898d0ac5ddb7cc567f7df6ac6/redirect"},"free_use_in_production":true,"feature":[]}
```

**Comment by HanSolo** [comment]
The new architectures are on purpose but the missing architectures of course not...I'm currently working on a process to find and update such packages automatically. Thx for the heads up 👍🏻

**Comment by marchof** [comment]
Maybe you can check the schema at the point in time when you add packages to the database (fail fast)?

**Comment by HanSolo** [comment]
Should be fixed with latest commit #a5bf096a2fac7b99e2b725e27d62c8eab8153542

**Comment by marchof** [comment]
Confirmed, thanks!
